pwa/app/lib/search/fuzzysortFilterer.ts

```diff
@@ -29,6 +29,17 @@ function searchEvents(
     limit,
     keys: ['key', 'name'],
     threshold: 0.5,
+    scoreFn: (r) => {
+      // For current_year events, return score * 2
+      // For current_year-1 events, return score * 1.95
+      // ...
+      // Down to score * 1, which will work for 20 years of events
+      const eventYear = Number.parseInt(r.obj.key.slice(0, 4));
+      const currentYear = new Date().getFullYear();
+      const yearDiff = currentYear - eventYear;
+
+      return r.score * Math.max(1, 2 - yearDiff / 20);
```

Comment:
is the 20 going to be an issue? should we make the denominator `currentYear - 1992` instead of something hardcoded?